src/components/LabelMenus/styles.module.css

```diff
@@ -1,64 +1,71 @@
+:root {
+  font-size: 16px;
+}
+
+.labelMenu {
+  width: 20%;
+}
+
 .labelMenus {
-  width: 291px;
-  height: 208px;
-  border: solid 1px #707070;
+  height: 13rem;
+  border: solid 0.0625rem #707070;
   background-color: #f4f4f4;
-  margin: 35px 40px;
-  padding: 30px 0px 0px 0px;
-  position: relative;
+  margin: 2.1875rem 0rem;
+  padding: 1.875rem 0rem 0rem 0rem;
+  /* position: relative; */
 }
 .highlightLabel {
   background-image: url("/highlightbar.png");
 
-  background-size: 8px 560px;
+  background-size: 0.5rem 35rem;
   background-repeat: no-repeat;
 }
 
 .lorem {
```

Comment:
rename to `.labelMenuDescription` or something like that to describe what will eventually go in this section